kqueen/engines/gce.py

```diff
@@ -118,6 +118,10 @@ def __init__(self, cluster, **kwargs):
                 'initialNodeCount': kwargs.get('node_count', 1),
                 'nodeConfig': {
                     'machineType': kwargs.get('machine_type', 'n1-standard-1')
+                },
+                'networkPolicy': {
+                    'provider': kwargs.get('network_provider', 'CALICO'),
+                    'enabled': kwargs.get('network_policy_enabled', True)
```

Comment:
So, when you pass `True` here, Calico Policy Engine will be automatically deployed into the cluster (from my understanding its k8s pod/service running inside the cluster) after its created. If you pass `False` instead, it won't be, but you can add it later with the `set_network_policy` method, is that correct? If so, can you create `unset_network_policy` which removes the policy as well?

Reply:
So by default it will be turned on, are you sure?)

Reply:
nono, its just draft) ill change it anyway to false by default, or refactor this dict-enabling method as @atengler said